provisioning/service/samples/createdelete_sample.js

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var assert = require('chai').assert;
+
+var provisioningClient = require('azure-iot-provisioning-service-client');
+
+var serviceClient = provisioningClient.DeviceEnrollment.fromConnectionString(process.argv[2]);
+
+var enrollment1 = {
+  registrationId: 'first',
+  attestation: {
+    type: 'tpm',
+    tpm: {
+      endorsementKey: 'a',
+      storageRootKey: 'b'
+    }
+  }
+};
+
+var enrollment2 = {
+  registrationId: 'second',
+  attestation: {
+    type: 'tpm',
+    tpm: {
+      endorsementKey: 'a',
+      storageRootKey: 'b'
+    }
+  }
+};
+
+serviceClient.createOrUpdateIndividualEnrollment(enrollment1, function(err, firstEnrollmentResponse, httpResponse) {
```

Comment:
>httpResponse [](start = 101, length = 12)

debugging purposes only, if we're not using it let's not advertise it.

Reply:
Fixed.

---
In reply to: [147065071](https://github.com/Azure/azure-iot-sdk-node/pull/121#discussion_r147065071) [](ancestors = 147065071)